src/olympia/devhub/templates/devhub/email/submission.txt

```diff
@@ -22,6 +31,6 @@ Add-ons help personalize the web experience for millions of users who have insta
 Happy developing!
 
 
--- 
+--
```

Comment:
SublimeText found and successfully exterminated a trailing space! 💪

Reply:
The standard for email signatures is 2 dashes and a trailing space though :)

Reply:
https://www.ietf.org/rfc/rfc3676.txt section 4.1

Reply:
🧐 But...the text version does not have it either...

Reply:
Changed.